update(UI overview): add bonita apps (#1362)
bonitasoft/bonita-doc: md/admin-application-process-list.md

# Process list in Bonita Administrator Application

> This page explains what a user with the _Administrator_ profile in Bonita Administrator Application can see and do about processes.

Reviewer: ```suggestion
This page explains what a user with the _Administrator_ profile can see and do about processes in [Bonita Administrator Application](admin-application-overview).
```

However, for future maintenance of the process definition, you are recommended to update all your process .bar files to the latest version each time you update your platform.

## Resolve a process
Resolving a process means completing the configuration in the Administrator Application and making sure that all dependencies are met. A process has to be resolved before it can be enabled. 
To resolve a process, you must complete the configuration of the actor mappings, parameters, forms, and connectors defined for the process.

* **Actors**: Every actor in a process must be mapped to at least one user, group, role or membership.
* **Parameters**: Every parameter defined in a process must have a value.
* **Connectors**: Every connector definition in a process must have a corresponding implemenetation (Class name).
* **Forms**: Every form defined in a process must be mapped to a page or a URL.
* Also, every business variable used in the process should have a corresponding model. 

To resolve a process for actors, parameters, connectors, and forms:
1. Go to _BPM_>_Processes_
2. On the row of the process, click on _View process details_ icon.

> 5. A summary of configuration problems is displayed below the process name in a well and warning icons are displayed in the left navigation list.
> 6. Modify your configuration for each element displayed.

Reviewer: ```suggestion
3. A summary of configuration problems is displayed below the process name in a well and warning icons are displayed in the left navigation list.
4. Modify your configuration for each element displayed.
```

4. In the opened modal window, type a new category name then press the _**Enter**_ key
5. Click on the _Save_ button.

After you created a category and added it to the process, you can add other processes to the category.

## Add a category to a process
1. Go to _BPM_>_Processes_ 
2. In the row of the process, click on the "View process details" icon
3. In _General_, click the _Pencil_ next to _Categories_ label
5. In the opened modal window, start typing the category name, and then select among the suggestions by using the arrow keys or mouse.
6. Press the _**Enter**_ key
6. Click on the _Save_ button

## Start a case for another user
This feature is available with the Enterprise, Performance, and Efficiency editions.  
To start a case for another user:
1. Go to _BPM_>_Processes_ 
2. In the row of the process, click on the "View more details" icon
3. Click on the _Start for_ button on the top right corner of the screen
4. In the popup, start typing the user for whom you are starting the case. Only valid users for the case are displayed.
5. Select the right user
6. Click on _OK_

The case is started as though the specified user had started it. 

> For example, if a you start a case for user A and a subsequent task is to be done by the manager of the user, it is assigned to user A's manager, not to your manager.

Reviewer: ```suggestion
For example, if you start a case for user A and a subsequent task is to be done by the manager of the user, it is assigned to user A's manager, not to your manager.
```

All of what follows belong to the [Live update feature](live-update.md) and are only available in for the Enterprise, Performance, and Efficiency editions.

## Edit the actor mapping

> You need to map more or less organization entities to an actor. To do so:

Reviewer: why "more or less" ? this is exactly that.
```suggestion
You need to map organization entities to an actor. To do so:
```

1. Go to _BPM_>_Processes_ 
2. In the row of the process, click on the "View more details" icon
3. Click on _Actors_ in the left menu.
To add entities:
4. Click the _+_ button in the user, group, role or membership column of the actor line.

> 5. In the opened modal window, click on the dropdown list to select one or several actors. The list displays the first five elements, then a number is displayed representing the other selected entities.

Reviewer: ```suggestion
5. In the opened modal window, click on the dropdown list to select one or several entities. The list displays the first five elements, then a number is displayed representing the other selected entities.
```

To remove entities:

> 6. In the opened modal window, there is a list of the actors already mapped. Click on the _X_ button next to the actor, or click on _Remove all_. A list appears filled with the actors you are about to remove. 

Reviewer: ```suggestion
6. In the opened modal window, there is a list of the entities already mapped with the actor. Click on the _X_ button next to an entity, or click on _Remove all_. A list appears filled with the entities you are about to remove. 
```

>    You can **undo** a removal by clicking on the _X_ button next to the actor or by clicking _Enable all_ 

Reviewer: ```suggestion
   You can **undo** a removal by clicking on the _X_ button next to the entity or by clicking _Enable all_ 
```

7. Click on the _Apply_ button.

::: info
**Note:** Only the first 200 actors are displayed in the dropdown.
::: 

## Define one or more Process managers
There are two stages to define a Process Manager:

1. Map organization entities to access the _Process Manager_ Portal to view information related to the processes they will be _Process Managers_ of.
   Only Administrators can do so.
    1. Go to _Organization_/_Profiles_ in the Administrator Application
    2. Choose the _Process manager_ profile. This will display the users, groups, roles, and memberships who already have this profile.
    3. Click on _More..._.
    4. Click on _Add a user_.
    5. Select a user by checking the box next to the user name.
    6. Click on _Add_. 
     The _Process manager_ profile is now assigned to the user.
     You can also assign a profile to a group, role, or membership.
     
2. Map the process to one or several entities that have the _Process Manager_ profile. 
    1. Go to _BPM_>_Processes_ 
    2. In the row of the process, click on the "View more details" icon. The Process managers currently assigned to the process are shown in the _Process manager mapping_ table.

>     4. To add a user, click the _Plus_ sign or _Pencil_ button in the _User_ column. A popup shows a list of the users who are currently process managers for this process.

Reviewer: ```suggestion
    3. To add a user, click the _Plus_ sign or _Pencil_ button in the _User_ column. A popup shows a list of the users who are currently process managers for this process.
```

>     5. Click _Select users_. A list of users with the Process manager profile is displayed.

Reviewer: No, it is not filtered. It displays all the users
```suggestion
    4. Click _Select users_. A list of users is displayed.
```

>     6. Select the user you want to be a Process manager for the process, and click on _Apply_. 

Reviewer: ```suggestion
    5. Select the user you want to be a Process manager for the process, and click on _Apply_. 
```

5. Browse to a .zip file containing the new connector implementation
6. Click on _Save_ to import the new implementation.

## Upload a new form
1. Go to _BPM_>_Processes_ 
2. In the row of the process, click on the "View more details" icon
3. Click on _Forms_ in the left menu
4. Click on the _Form list_ tab. A list of form is displayed. These forms are only visible to the current process. 
5. Click the _plus_ button at the bottom of the list. A file selector popup is displayed. 
6. Browse to a .zip containing a form.
7. Click on _Next_, then if your form requires some authorization, it will be displayed.
8. Click on _Confirm_.

## Edit an existing form
1. Go to _BPM_>_Processes_ 
2. In the row of the process, click on the "View more details" icon
3. Click on _Forms_ in the left menu
4. Click on the _Form list_ tab. A list of forms is displayed. These forms are only visible to the current process.
5. Click the _Pencil_ button of a form line. A file selector popup is displayed.
6. Browse to a .zip containing a form
7. Click _Next_, then if your form requires some authorization, it will be displayed.
8. Click on _Confirm_.

> ## Fix forms

Reviewer: ```suggestion
## Form mapping of a task, case overview, or process instantiation
```

Author: I will continue in the trend "verb + object" => Map forms with process instantiation, tasks, or case overview

1. Go to _BPM_>_Processes_ 
2. In the row of the process, click on the "View more details" icon
3. Click on _Forms_ in the left menu.

> 5. The tables are displayed which list available forms. Click on a red link. A field input appears.
> 6. Starting to type some text, and any matching names of installed forms will be proposed. If there is no match for the name you enter, it will be considered as a URL.
> 7. Click the _Tick_ button to validate your change or _**X**_ to dismiss your change.

Reviewer: ```suggestion
4. The tables are displayed which list available forms. Click on a red link. A field input appears.
5. Starting to type some text, and any matching names of installed forms will be proposed. If there is no match for the name you enter, it will be considered as a URL.
6. Click the _Tick_ button to validate your change or _**X**_ to dismiss your change.
```

::: info

> **Note:** You can also upload a form in order to create a new mapping.

Reviewer: ```suggestion
**Note:** You can also upload a new form in order to create a new form mapping (see the above section Upload a new form).
```
